### PR TITLE
Wireshark

### PR DESCRIPTION
**Pull request by Fred-Barclay**
This will probably need to to tweaked over time, but here's a basic wireshark profile (partly handles #952).

### CODE REVIEW

**Comment by netblue30** [comment]
All merged, thanks.

**Comment by norpol** [comment]
Hi, any update regarding this?

**Comment by netblue30** [comment]
Yes, it is all in git in this moment.

File: platform/debian/conffiles [hunk 1]
```diff
@@ -27,6 +27,7 @@
 /etc/firejail/conkeror.profile
 /etc/firejail/corebird.profile
 /etc/firejail/cpio.profile
+/etc/firejail/cryptocat.profile
```
Comment (Fred-Barclay) [inline platform/debian/conffiles]: Hi @norpol Actually, it's a bit more complicated than that. :smile:
 - For a previous pull request, I'd added `/etc/Cryptocat.profile` and `/etc/cryptocat.profile`. The lower-case profile is there to ensure compatibility, since some distros or users may create a cryptocat launcher (with a lowercase `c`). 

`/etc/cryptocat.profile` only points to `/etc/Cryptocat` and is not an independent profile:
```
$ cat /etc/firejail/cryptocat.profile
include /etc/Cryptocat.profile
```
Other profiles, such as Cyberfox or Mathematica, do the same thing.

- In that previous pull request, I'd neglected to add `/etc/cryptocat.profile` to `/platform/debian/conffiles,` though I did add `/etc/Cryptocat.profile`

- So for this pull request, I **added** `/etc/cryptocat.profile` to `/platform/debian/conffiles`. The `C` in fact was not changed to `c`; I just added an additional line so both `/etc/Cryptocat.profile` and `/etc/cryptocat.profile` are in here. Nothing was renamed, only another line added. :) 
https://github.com/netblue30/firejail/blob/master/platform/debian/conffiles#L31

TL;DR,
The `C` was not changed to `c`, but a new profile was added to guarantee compatibilty in the future.

Cheers!
Fred

Comment (norpol) [inline platform/debian/conffiles]: Thank you!

Comment (Fred-Barclay) [inline platform/debian/conffiles]: No worries, mate. :smiley:
